controllers.js

```diff
@@ -1,10 +1,20 @@
-const {getTopicModel} = require('./models')
+const {
+    getTopicModel,
+    getArticleModel
+} = require('./models')
 
 exports.getTopic = (req, res, next) => {
     getTopicModel()
     .then((topics) => {
         res.status(200).send(topics);
     })
-    .catch(next)
+    .catch(next);
+}
 
+exports.getArticle = (req, res, next) => {
+    getArticleModel()
+    .then((articles) => {
+        res.status(200).send(articles);

```

Comment:
unfortunately this does still need to be changed - the response should be an object with a key containing the data, i.e. `.send({ articles: articles });`  which can be shortened to  `.send({ articles });`